apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ClientCacheExtensions.kt

```diff
@@ -100,88 +99,46 @@ fun ApolloClient.Builder.store(store: ApolloStore, writeToCacheAsynchronously: B
       .writeToCacheAsynchronously(writeToCacheAsynchronously)
 }
 
-/**
- * Use with [watch] to configure how errors are propagated.
- * Note that any non-Apollo exceptions (e.g. `OutOfMemoryError`) will still be thrown regardless of the policy.
- */
-enum class WatchErrorHandling {
-  ThrowCacheErrors,
-  ThrowNetworkErrors,
-  ThrowAll,
-  IgnoreErrors,
-}
-
 /***
  * Gets the result from the network, then observes the cache for any changes.
  * Overriding the [FetchPolicy] will change how the result is first queried.
- * Exception are ignored by default, this can be changed by setting [fetchErrorHandling] for the first fetch and [refetchErrorHandling]
- * for subsequent fetches.
+ * Network and cache exceptions are ignored by default, this can be changed by setting [fetchThrows] for the first fetch and [refetchThrows]
+ * for subsequent fetches (non Apollo exceptions like `OutOfMemoryError` are always propagated).
```

Comment:
Nit pick. Use `@param`? It duplicates some content but from a consumer point of view, it's easier to find the information?

```kotlin
/**
 * Gets an initial result then observes the cache for any changes.
 * Use [fetchPolicy] to control the initial fetch and [refetchPolicy] for the subsequent ones.
 * 
 * Network and cache exceptions are ignored by default, this can be changed by setting [fetchThrows] for the first fetch and [refetchThrows]
 * for subsequent fetches (non Apollo exceptions like `OutOfMemoryError` are always propagated).
 *
 * @param fetchThrows whether to throw if an [ApolloException] happens during the initial fetch. Default: false
 * @param refetchThrows whether to throw if an [ApolloException] happens during a refetch. Default: false
 */
```